docs/learn/learn-staking.md

```diff
@@ -623,7 +623,7 @@ in the percentage of staking rewards that go to the Treasury.
 - **Blue line**: Inflation rewards to stakers
 - **Green line**: Staker rate of return
```

Comment:
```suggestion
- **Green line**: Annual interest rate for stakers, i.e. portion of rewards over the total amount of staked tokens.
```

Reply:
What do you think of adding a numeric example for better understanding for a generic reader?